packages/react-paypal-js/src/v6/hooks/usePayLaterOneTimePaymentSession.ts

```diff
@@ -85,9 +98,14 @@ export function usePayLaterOneTimePaymentSession({
         } as PayPalPresentationModeOptions;
 
         if (createOrder) {
-            await sessionRef.current.start(startOptions, createOrder());
+            const result = await sessionRef.current.start(
+                startOptions,
+                createOrder(),
+            );
+            return result;
         } else {
-            await sessionRef.current.start(startOptions);
+            const result = await sessionRef.current.start(startOptions);
+            return result;
```

Comment:
We have this setup in every session hook, but I believe this could be shortened:

```suggestion
const result = await sessionRef.current.start(
    startOptions,
    createOrder?.()
);
return result;
```

cc @EvanReinstein should we do this with other session hooks as well?